File: C/exo3_1.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    int a, b;
    if(argc!=3)     //si la personne n'a pas entré deux arguments
    {
        printf("Erreur! Deux nombres requises!");
        exit(1);
    }
    a=atoi(argv[1]);
    b=atoi(argv[2]);
    printf("Somme: %d\n", a+b);
    return 0;
}

File: python/exo3_1.py
import sys

if len(sys.argv) != 3 :     #si la personne n'a pas entré deux arguments
        print("Erreur! Deux nombres requises!")
        sys.exit(1)

a = int(sys.argv[1])
b = int(sys.argv[2])
print(f"Somme: {a+b}")